modules/serverless-sinkbinding-kn.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// serverless/knstive_eventing/serverless-sinkbinding.adoc
+
+[id="serverless-sinkbinding-kn_{context}"]
+= Using SinkBinding with the Knative CLI (`kn`)
+
+ This guide describes the steps required to create, manage and delete a SInkBinding instance using `kn` commands.
+
+.Prerequisites
+
+* You will need to have a Knative Serving and Eventing installation.
+* You will need to have the `kn` CLI installed.
+
+.Procedure
+
+. To check that SinkBinding is set up correctly, create a Knative event display service, or event sink, that dumps incoming messages to its log.
++
+You can create the service by entering the following command:
++
+----
+$ kn service create event-display --image quay.io/openshift-knative/knative-eventing-sources-event-display
+----
+
+. Create a SinkBinding that will direct events to the service.
++
+You can create the SinkBinding by entering the following command:
++
+----
+$ kn source sinkbinding create testevents --sink broker:default --resource "event:v1" --service-account events-sa
+----
+
+. Create a trigger from the `default` broker, to filter events to the service created in the previous step, by entering the following command:
++
+----
+$ kn trigger create event-display-trigger --broker default --sink svc:event-display \ source=dev.knative.samples/testevents \
```

Comment:
this should be probably `--filter source=dev.knative.samples/testevents` but the value totally depends on the CloudEvents that the Deployment mentioned above is emitting.

Reply:
Removed the trigger part altogether since it's not relevant?